src/entities/applet/model/slice.ts

```diff
@@ -1,11 +1,11 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
 import {
-  FlowProgress,
   ActivityPipelineType,
   EntitiesInProgress,
+  FlowProgress,
   ProgressPayload,
-} from '../lib';
+} from '@app/abstract/lib/types';
```

Comment:
Please create index.ts files in app/abstract and reexport types and/or utils so that it will be possible to import anything from the module. For example
`import { FlowProgress } from "@app/abstract/lib"`